### PR TITLE
[Plugin] added component ID for Python aiohttp

### PR DESCRIPTION
**Pull request by tom-pytel**
Added component ID for Python plugin for module `aiohttp`.

### Add an agent plugin to support Python aiohttp module
- [x] Add a test case for the new plugin, refer to [the doc](https://github.com/apache/skywalking/blob/master/docs/en/guides/Plugin-test.md)
- [x] Add a component id in [the component-libraries.yml](https://github.com/apache/skywalking/blob/master/oap-server/server-bootstrap/src/main/resources/component-libraries.yml)
- [x] Add a logo in [the UI repo](https://github.com/apache/skywalking-rocketbot-ui/tree/master/src/views/components/topology/assets)

- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

**Review by kezhenxu94 — DISMISSED** [review]
Thanks

**Review by wu-sheng — COMMENTED** [review]
I think changlog should be updated as new component id added.

**Comment by tom-pytel** [comment]
> I think changlog should be updated as new component id added.

Done

File: CHANGES.md [hunk 1]
```diff
@@ -14,6 +14,9 @@ Release Notes.
 * Support collecting logs of log4j, log4j2, and logback in the tracing context with a new `logger-plugin`.
 * Fix the unexpected RunningContext recreation in the Tomcat plugin.
 
+#### Python Agent
+* Added plugin for aiohttp module requester and server.
```
Comment (wu-sheng) [inline CHANGES.md:18]: This changlog belongs to the OAP-backend. 
And added->add.

Comment (tom-pytel) [reply CHANGES.md:18]: I am confused, so what do you want me to add and where?

Comment (kezhenxu94) [reply CHANGES.md]: > I am confused, so what do you want me to add and where?

Done for you

Comment (wu-sheng) [reply CHANGES.md]: Sorry, review from cell phone, was trying to type less words.
1. Move the change into OAP-backend part, as the file you changed belongs there. 
2. As this
```suggestion
* Add component id for Python aiohttp plugin.
```

The changelog file in the main repo is tracking the its own change. New python plugin belongs to skywalking-python repo's changelog.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**